graphql_api/types/commit/commit.graphql

```diff
@@ -4,17 +4,22 @@ type Commit {
   createdAt: DateTime!
   commitid: String!
   author: Owner
+  # totals to be removed with #2286
   totals: CoverageTotals
   parent: Commit
   pullId: Int
   branchName: String
   yaml: String
   yamlState: YamlStates
   ciPassed: Boolean
+  # flagNames to be removed with #2286
   flagNames: [String]
+  # coverageFile to be removed with #2286
   coverageFile(path: String!, flags: [String], components: [String]): File
   compareWithParent: ComparisonResult
```

Comment:
Do you know if this `ComparisonResult` is a coverage concept? If so, we can nest it (can do it in a separate PR/ticket as needed)

Reply:
we use it for the commit detail page so i think it can stay in top level commit for now

Reply:
Ah nice got it, sounds good!